src/ModulusRemainder.cpp

```diff
@@ -213,6 +214,12 @@ void ComputeModulusRemainder::visit(const Shuffle *op) {
     result = ModulusRemainder{};
 }
 
+void ComputeModulusRemainder::visit(const VectorInstruction *op) {
+    internal_error << "modulus_remainder of VectorInstruction:\n"
```

Comment:
This seems likely to trigger because ModulusRemainder is used inside codegen for alignment queries.

Reply:
Copy what the VectorReduce visitor does